packages/dds/tree/src/edit-manager/editManager.ts

```diff
@@ -61,6 +63,17 @@ export class EditManager<
         super();
     }
 
+    public setMinimumSequenceNumber(minimumSequenceNumber: number): void {
+        this.minimumSequenceNumber = minimumSequenceNumber;
```

Comment:
It would be good to add an assert to verify the new `minimumSequenceNumber` is >= to the previous one.

Reply:
Added assert statement